src/unix/core.c

```diff
@@ -2024,10 +2024,10 @@ unsigned int uv_available_parallelism(void) {
 #ifdef __linux__
   {
     double rc_with_cgroup;
-    uv__cpu_constraint c = {0, 0, 0.0};
+    uv__cpu_constraint c = {0, 0, 0, 0.0};
 
-    if (uv__get_constrained_cpu(&c) == 0 && c.period_length > 0) {
-      rc_with_cgroup = (double)c.quota_per_period / c.period_length * c.proportions;
+    if (uv__get_constrained_cpu(&c) == 0 && c.quota > 0) {
+      rc_with_cgroup = (double)c.quota;
```

Comment:
I suggest getting rid of `rc_with_cgroup` and just use `c.quota` directly. All this type casting is superfluous and confusing at best, UB at worst (if `c.quota` cannot be represented as `double`)

Reply:
Done!